ArcadiaParties-SPA/src/app/app.component.ts

```diff
@@ -1,20 +1,15 @@
 import { Component } from '@angular/core';
-import { select, Store } from '@ngrx/store';
-import { Observable } from 'rxjs';
+import { Store } from '@ngrx/store';
 
 import { State } from './reducers/reducers';
-import { selectTitle } from './selector/selector';
-import { changeTitleAction } from './actions/actions';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
-  title$: Observable<string> = this.store.pipe(select(selectTitle));
 
   constructor(private store: Store<State>) {
```

Comment:
Why do you need `Store` if you don't use it?

Reply:
It deleted